### ISSUE TITLE
Duplicate permissions on lift

### ISSUE THREAD

**Issue by dottodot** [tjwebb/sails-permissions#44]
I'm still having issues similar to #27 but it only seems to be permissions now. So every time I lift a permission entry is created for each model and action for the admin role.

**Comment by tjwebb** [comment]
The default permissions are added using `findOrCreate`. Does it keep adding more and more copies of permissions every time you lift? i.e. if you lift 10 times, do you see 10 copies?

**Comment by dottodot** [comment]
Yes that seems to what's happening. If I delete all permissions then lift it creates 31 entries, then on the next lift there are 62 entries and so on.

**Comment by tjwebb** [comment]
Ok I think this is because the findOrCreate criteria isn't reliably treating the permissions as unique; this shouldn't impede functionality, and is basically a storage leak. I'll try to fix soon.

**Comment by dottodot** [comment]
I've found that in fixtures by changing the findOrCreate to what's below fixes it's, although not sure if it will cause other issues.

```
Permission.findOrCreate({model:newPermission.model, action: newPermission.action, role: newPermission.role}, newPermission);
```
```
Permission.findOrCreate({model:permission.model, action: permission.action, role: permission.role}, permission);
```

**Comment by tjwebb** [comment]
I think the main problem is that Permission has no unique field besides the primary key, so there's no way for findOrCreate to work reliably in this context. I'm working on a fix now.

**Comment by tjwebb** [comment]
@dottodot I've made some improvements to the fixtures. Are you still seeing this issue on the latest release?

**Comment by dottodot** [comment]
Is the new release available as I can only get 1.0.19 which has the same issue

**Comment by dottodot** [comment]
OK just noticed the update is now available and it now longer seems to be creating duplicates.
